pkg/roachprod/install/cockroach.go

```diff
@@ -817,24 +814,28 @@ func (c *SyncedCluster) createFixedBackupSchedule(
 	}
 	l.Printf("%s: creating backup schedule", c.Name)
 	auth := "AUTH=implicit"
-
 	collectionPath := fmt.Sprintf(`%s/roachprod-scheduled-backups/%s/%v?%s`,
 		externalStoragePath, c.Name, timeutil.Now().UnixNano(), auth)
 
 	// Default scheduled backup runs a full backup every hour and an incremental
 	// every 15 minutes.
-	scheduleArgs := `RECURRING '*/15 * * * *'
-FULL BACKUP '@hourly'
-WITH SCHEDULE OPTIONS first_run = 'now'`
-
+	scheduleArgs := `RECURRING '*/15 * * * *' FULL BACKUP '@hourly' WITH SCHEDULE OPTIONS first_run = 'now'`
 	if scheduledBackupArgs != "" {
 		scheduleArgs = scheduledBackupArgs
 	}
-
-	createScheduleCmd := fmt.Sprintf(`-e
-CREATE SCHEDULE IF NOT EXISTS test_only_backup FOR BACKUP INTO '%s' %s`,
+	createScheduleCmd := fmt.Sprintf(`CREATE SCHEDULE IF NOT EXISTS test_only_backup FOR BACKUP INTO '%s' %s`,
 		collectionPath, scheduleArgs)
-	return c.ExecSQL(ctx, l, "" /*tenantName*/, []string{createScheduleCmd})
+
+	node := Node(1)
+	binary := cockroachNodeBinary(c, node)
+	url := c.NodeURL("localhost", c.NodePort(node), "" /* tenantName */)
+	fullCmd := fmt.Sprintf(`COCKROACH_CONNECT_TIMEOUT=0 %s sql --url %s -e %q`,
+		binary, url, createScheduleCmd)
+	// Instead of using `c.ExecSQL()`, use the more flexible c.Run(), which allows us to
+	// 1) prefix the schedule backup cmd with COCKROACH_CONNECT_TIMEOUT=0.
+	// 2) run the command against the first node on the cluster.
+	l.Printf("%s\n", fullCmd)
```

Comment:
I don't think you intended to commit this?

Reply:
oops. resolved.